### DISCUSSION TITLE
About transport class

### DISCUSSION THREAD

**Discussion by wuhaibin001** [EIPStackGroup/OpENer#480] [Q&A]
Hi,
  I have a question I don't quite understand, I would like to consult. cipioconnection.c file，SendConnectedData function，why use Sequenced Address Items if not Connection Class 0？《CIP Networks Library Volume 2：EtherNet/IP Adaptation of CIP》(Edition 1.21 April 2016)，in section 2-6.2.3, indicate that Sequenced Address Item shall be used for CIP transport class 0 and class 1 connected data.

**Comment by MartinMelikMerkumians** [comment]
Sorry, there is no 2-6.2.3 in the current specification.

Also OpENer does not support Class 0 communication. Perhaps its possible, but its neither in the EDS file, nor actively testes
